src/index.js

```diff
@@ -149,7 +146,7 @@ const initialize = async () => {
   }
 
   const updateButtons = () => {
-    const accountButtonsDisabled = !isMetaMaskInstalled() || !isMetaMaskConnected()
+    const accountButtonsDisabled = !MetaMaskOnboarding.isMetaMaskInstalled() || !isMetaMaskConnected()
```

Comment:
`isMetaMaskInstalled` is already in scope here (brought in on line 40), so there's no need to reference it via the `MetaMaskOnboarding` class. This line can remain unchanged.

```suggestion
    const accountButtonsDisabled = !isMetaMaskInstalled() || !isMetaMaskConnected()
```